[docs] Document accepting Convex invites with a different verified Expo email (#48207)
expo/expo: docs/pages/guides/using-convex.mdx

/>

If you remove the link with `eas integrations:convex:project:delete` or `eas integrations:convex:team:delete`, EAS removes its integration metadata. These commands do not destroy resources on Convex.

## Troubleshooting

### Accept a team invite sent to a different email

> The Convex team invitation goes to the verified email on your Expo account, but Convex only supports signing in with Google or GitHub. If the email of the account you sign in with does not match the invited email, the invitation page shows an error saying the invite belongs to another email address. Navigating away from that page leaves you in a separate Convex account that does not contain your project.

Reviewer: ⚠️ **correctness** `prose.md#core-principles`

"the invite belongs to another email address" does not match the screenshot right below, where Convex says the email address has not been added to your account. Use the message the reader sees, which also trims this 30-word sentence.

```suggestion
The Convex team invitation goes to the verified email on your Expo account, but Convex only supports signing in with Google or GitHub. If your Google or GitHub email is different, the invitation page says the invited email address has not been added to your Convex account. Navigating away from that page leaves you in a separate Convex account that does not contain your project.
```

<!-- docs-review -->
